padd.sh

```diff
@@ -41,6 +41,7 @@ magenta_text="${CSI}95m" # Magenta
 cyan_text="${CSI}96m"    # Cyan
 reset_text="${CSI}0m"    # Reset to default
 clear_line="${CSI}0K"    # Clear the current line to the right to wipe any artifacts remaining from last print
+clear_cursor="${CSI}1K"  # Clear the current line to from the beginning through the cursor
```

Comment:
```suggestion
clear_cursor="${CSI}1K"  # Clear the current line from the beginning through the cursor
```

Is this variable used?

Reply:
No, it's a leftover. I'll push force when I've worked out an edge-case bug I just noticed  ;-)